### PR TITLE
Docs: Fix options for os_image and link to CLUO

### PR DESCRIPTION
**Pull request by indradhanush**

### CODE REVIEW

**Review by invidian — APPROVED** [review]
LGTM

File: docs/flatcar-linux/aws.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ Reference the DNS zone id with `"${aws_route53_zone.zone-for-clusters.zone_id}"`
 | worker_count | Number of workers | 1 | 3 |
 | controller_type | EC2 instance type for controllers | "t3.small" | See below |
 | worker_type | EC2 instance type for workers | "t3.small" | See below |
-| os_image | AMI channel for a Flatcar Linux derivative | flatcar-stable | coreos-stable, coreos-beta, coreos-alpha, flatcar-stable, flatcar-beta, flatcar-alpha |
+| os_image | AMI channel for a Flatcar Linux derivative | flatcar-stable | flatcar-stable, flatcar-beta, flatcar-alpha |
```
Comment (rata) [inline docs/flatcar-linux/aws.md]: s/derivative//?

File: docs/flatcar-linux/bare-metal.md [hunk 2]
```diff
@@ -360,7 +360,7 @@ Check the [variables.tf](https://github.com/kinvolk/lokomotive-kubernetes/blob/m
 |:-----|:------------|:--------|
 | cluster_name | Unique cluster name | mercury |
 | matchbox_http_endpoint | Matchbox HTTP read-only endpoint | http://matchbox.example.com:port |
-| os_channel | Channel for a Flatcar Linux derivative | coreos-stable, coreos-beta, coreos-alpha, flatcar-stable, flatcar-beta, flatcar-alpha |
+| os_channel | Channel for a Flatcar Linux derivative | flatcar-stable, flatcar-beta, flatcar-alpha |
```
Comment (rata) [inline docs/flatcar-linux/bare-metal.md]: Idem

**Review by rata — COMMENTED** [review]
LGTM. Not sure we want to keep the "derivative", I guess we don't

File: docs/flatcar-linux/aws.md (hunk 1 above)
Comment (invidian) [reply docs/flatcar-linux/aws.md]: Good catch :+1:

Comment (indradhanush) [reply docs/flatcar-linux/aws.md]: Good catch. Will fix.

**Review by rata — APPROVED** [review]

**Pull request merged**